### PR TITLE
Fix iOS example in CI

### PR DESCRIPTION
**Pull request by simlay**
I found the fix in https://github.com/TimNN/cargo-lipo/issues/41#issuecomment-758043784.

I'm not in love with this fix.

### CODE REVIEW

**Comment by est31** [comment]
It still errors:

```
ld: in target/universal/debug/libcpal_ios_example.a(cpal_ios_example-84f53eaf88c5c2cf.2wd2snyha1ct501c.rcgu.o), building for iOS Simulator, but linking in object file built for iOS, for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by est31** [comment]
@simlay friendly bump, want to fix the error?

File: examples/ios-feedback/cpal-ios-example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -253,6 +253,7 @@
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=*]" = arm64;
```
Comment (simlay) [inline examples/ios-feedback/cpal-ios-example.xcodeproj/project.pbxproj:256]: https://stackoverflow.com/a/63955114

**Comment by simlay** [comment]
> @simlay friendly bump, want to fix the error?

Should be good!

**Review by est31 — APPROVED** [review]
Thanks!

**Pull request merged**